### ISSUE TITLE
Research-Oriented Inquiry on DVMark's Isolated Video Watermark Training

### ISSUE THREAD

**Issue by Anthony-0721** [facebookresearch/videoseal#34]
Dear Facebook Team,

  As a graduate student in Network & Information Security, I'm conducting research on digital watermark robustness.And I want to reproduce the paper DVMarkA_Beep_Multiscale_frame_for_Video_Watermarking. Your framework shows exceptional potential for my thesis on 【对抗性视频水印】. While my coding skills are developing, I can provide comprehensive experimental data to support debugging efforts.

Key Requests:
1. 【Configuration Guidance】
   - How to modify 【configs】 to disable 【images modules】 while keeping 【watermark encoder/decoder】 active?
2. 【Training Protocol】
   - Should I use the existing 【train.py】 flag or create a dedicated 【train_videos_watermark.py】?
   - 
                                                                                                                          Best regards,
                                                                                                                          Mengqi Jiao
                                                                                                                          Master Candidate in Network Security
                                                                                                                          GZNU

**Comment by pierrefdz** [comment]
- you can use any 3d architecture you want, and you'll have to construct the new embedder/ extractor in https://github.com/facebookresearch/videoseal/blob/main/videoseal/models/embedder.py and https://github.com/facebookresearch/videoseal/blob/main/videoseal/models/extractor.py
- Make sure to set batch_size_video and videoseal_step_size to 1.
- Here is an example of command: 
`--balanced False --total_gnorm 1.0 --scaling_w_schedule Cosine,scaling_min=0.016,start_epoch=200,epochs=200 --scaling_w 0.2 --attenuation None --scaling_i 1.0 --nbits 64 --lambda_dec 1.0 --lambda_det 0.0 --lambda_d 0.0 --lambda_i 0.0 --perceptual_loss yuv --seed 0 --scheduler CosineLRScheduler,lr_min=1e-6,t_initial=301,warmup_lr_init=1e-8,warmup_t=10 --optimizer AdamW,lr=1e-4 --saveimg_freq 50 --eval_freq 10 --full_eval_freq 50 --batch_size_video_eval 1 --batch_size_video 1 --batch_size_eval 16 --batch_size 16 --iter_per_valid 10 --frames_per_clip 16 --videowam_step_size 1 --finetune_detector_start 2000 --epochs 301 --iter_per_epoch 1000 --img_size_proc 256 --img_size_val 256 --img_size 256 --disc_in_channels 1 --disc_start 50 --hidden_size_multiplier 1 --augmentation_config configs/all_augs_v3.yaml --video_start 0 --prop_img_vid 0 --workers 0 --extractor_model convnext_tiny_temporal --embedder_model unet_small2_yuv_quant_conv3d --video_dataset sa-v --image_dataset none`
